### DISCUSSION TITLE
Capacitor with SAPUI5

### DISCUSSION THREAD

**Discussion by izeryab** [ionic-team/capacitor#4627] [General] [answered]
Hello great people!

I am working on a project in which i am using capacitor with sapUI5 but i find it really painful because i didn't found any help or sample code of how to use basic features however i managed to start with capacitor 2 but now with capacitor 3 i am stuck and kind find out what's wrong.
Is there any good source or sample implementation of basic features of capacitor like camera access?

**Comment by robingenz** [comment, marked as answer]
Take a look at the documentation of the camera plugin: https://capacitorjs.com/docs/apis/camera

**Reply by izeryab** [reply]
@robingenz thank you but i have gone through this and but i still cannot import Camera thats why i am wondering if there is any complete capacitor sample application so i can just download and try to run and then try to compare with my app and check what i am doing wrong. like this one https://github.com/robingenz/capacitor-plugin-demo

**Reply by robingenz** [reply]
You can try the Capacitor testapp: https://github.com/ionic-team/capacitor-testapp
